File: NET/Kata.RockPaperScissor/Paper.cs
using System;
using System.Collections.Generic;
using System.Linq;
using System.Text;
using System.Threading.Tasks;

namespace Kata.RockPaperScissor
{
    public class Paper : Shape
    {
        public Paper()
        {
            ShapeType = RockPaperScissor.ShapeType.Paper;
            Name = "Paper";
            winsTo = new List<ShapeType>() { ShapeType.Rock, ShapeType.Spock };
        }
    }
}

File: JavaScript/src/objects/paper.js
var Shape = require('./shape');

var Paper = Object.create(Shape);

Paper.setup = function (ShapeType) {
	this.init({
		shapeType: ShapeType.paper,
		winsTo: [ShapeType.rock, ShapeType.spock]
	});
}

module.exports = Paper;